app/Http/Controllers/UserController.php

```diff
@@ -11,7 +11,7 @@ public function update(User $user, UpdateUserRequest $request)
     {
         // TASK: change this line to not allow is_admin field to be updated
         // Update only the fields that are validated in UpdateUserRequest
-        $user->update($request->all());
+        $user->update($request->except(['is_admin']));
```

Comment:
may be this not secure enough. my solution is this 
$user->update($request->validated());

It's my opinion but you doing right, keep going.